ci-operator/step-registry/vsphere/deprovision/usertags/vsphere-deprovision-usertags-ref.yaml

```diff
@@ -0,0 +1,15 @@
+ref:
+  as: vsphere-deprovision-usertags
+  from: upi-installer
+  cli: latest
```

Comment:
In this step, I think we do not need the oc cli

Reply:
ok